module/patches/getTemplate.mjs

```diff
@@ -21,7 +21,7 @@ export default function setupDisableTemplateCache() {
       delete _templateCache[path];
     }
 
-    return _getTemplate(path);
+    return _getTemplate(path, id);
```

Comment:
```suggestion
    return _getTemplate(path, ...args);
```
might this be more futureproof?
not sure how this responds when there are no further args